Source/Core/Core/Debugger/PPCDebugInterface.h

```diff
@@ -76,6 +76,7 @@ class PPCDebugInterface final : public Common::DebugInterface
   void RunToBreakpoint() override;
   int GetColor(unsigned int address) override;
   std::string GetDescription(unsigned int address) override;
+  u32 GetMemoryAddressFromInstruction(std::string instruction);
```

Comment:
This can likely be a const qualified member function.

Reply:
I had mean that the function can be const qualified, like:

```cpp
u32 GetMemoryAddressFromInstruction(std::string instruction) const;
```

not like:

```cpp
const u32 GetMemoryAddressFromInstruction(std::string instruction);
```
(which is qualifying the return value, not the function)